### PR TITLE
Add reflection to relay

### LINKED ISSUE #164: Add Server reflection to relay
To make it more convenient to the users of `grpcurl` we can add [server reflection](https://github.com/fullstorydev/grpcurl#server-reflection) to the relay.
Than no one needs to point to the protocol file `-import-path ./tools/invoker/proto -proto helloworld.proto`.
See [here](https://github.com/ease-lab/vSwarm/blob/main/benchmarks/aes/go/server.go) how to implement

### PR DESCRIPTION
**Pull request by harshitgarg22**
Fixes #164

### CODE REVIEW

**Review by dhschall — APPROVED** [review]
Looks good 👍

**Review by dhschall — DISMISSED** [review]

**Review by dhschall — COMMENTED** [review]
The context is not yet merged and tagged in vSwarm-proto
I think we should merge this this first and then fix the 0.1.4 version here and then proceed with this PR

File: utils/tracing/integ-tests/client-server/helloworld.proto [hunk 1]
```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```
Comment (dhschall) [inline utils/tracing/integ-tests/client-server/helloworld.proto:1]: Why is this file new?

**Review by dhschall — APPROVED** [review]

File: utils/tracing/integ-tests/client-server/helloworld.proto (hunk 1 above)
Comment (harshitgarg22) [reply utils/tracing/integ-tests/client-server/helloworld.proto:1]: It wasn't there yeah, and the generated proto file was just there. If someone later on looks at the generated proto file its not clear how it was generated, which is why I put it there.

**Pull request merged**